main_f4.c

```diff
@@ -431,6 +550,12 @@ main(void)
 		if (board_test_force_pin())
 			continue;
 
+#if INTERFACE_USART
+		/* if the USART port RX line is still receiving a break, just loop back */
+		if (board_test_usart_receiving_break())
```

Comment:
If the USART port is still receiving a break, loop back like if the force-bl pins were strapped. Do you want this here?